### PR TITLE
media-libs/mesa-13.0.0: add libressl support

### PR DESCRIPTION
**Pull request by Wonko7**
This enables compiling with libressl instead of openssl. 
Let me know if you want me to change something.

### CODE REVIEW

File: media-libs/mesa/mesa-13.0.0.ebuild [hunk 1]
```diff
@@ -104,7 +104,10 @@ RDEPEND="
 	nettle? ( dev-libs/nettle:=[${MULTILIB_USEDEP}] )
 	!nettle? (
 		gcrypt? ( dev-libs/libgcrypt:=[${MULTILIB_USEDEP}] )
-		!gcrypt? ( dev-libs/openssl:=[${MULTILIB_USEDEP}] )
+		!gcrypt? (
+			libressl? ( dev-libs/libressl:=[${MULTILIB_USEDEP}] )
+			!libressl? ( dev-libs/openssl:=[${MULTILIB_USEDEP}] )
```
Comment (SoapGentoo) [inline media-libs/mesa/mesa-13.0.0.ebuild]: both libressl and openssl need `:0=`

File: media-libs/mesa/mesa-13.0.0.ebuild [hunk 2]
```diff
@@ -44,11 +44,11 @@ done
...
 IUSE="${IUSE_VIDEO_CARDS}
 	bindist +classic d3d9 debug +dri3 +egl +gallium +gbm gcrypt gles1 gles2
-	+llvm nettle +nptl opencl osmesa pax_kernel openmax +openssl pic selinux
-	vaapi valgrind vdpau wayland xvmc xa kernel_FreeBSD"
+	libressl +llvm nettle +nptl opencl osmesa pax_kernel openmax +openssl pic
+	selinux vaapi valgrind vdpau wayland xvmc xa kernel_FreeBSD"
 
 REQUIRED_USE="
-	|| ( gcrypt nettle openssl )
+	|| ( gcrypt nettle openssl libressl )
```
Comment (SoapGentoo) [inline media-libs/mesa/mesa-13.0.0.ebuild]: remove libressl here again. `USE="libressl"` does not unconditionally request Libressl.

Comment (Wonko7) [inline media-libs/mesa/mesa-13.0.0.ebuild]: But that means I can't build this with USE="libressl -openssl", right? Just double checking, I didn't know the openssl flag was required for using libressl.

**Comment by SoapGentoo** [comment]
@Wonko7 please rebase on 13.0.1

**Comment by monsieurp** [comment]
@gentoo/x11

**Comment by a17r** [comment]
Please rebase on 13.0.2 and also add it to 9999.

**Comment by mgorny** [comment]
I'm sorry but this doesn't really look mergeable as-is, and there has been no activity from the submitter since Nov 2016. I'm going to close it. Feel free to submit a new pull request once you implement the requested changes (i.e. update it).

**Comment by mattst88** [comment]
I guess I didn't see this or something. mesa-13.0.x has libressl support since commit c941486be4e224930cdaca7cca74449bef07d1e7b (but the cited bug number is wrong)

**Pull request closed without merging**